index.d.ts

```diff
@@ -1,33 +1,5 @@
 declare module '@articulate/authentic' {
-  interface VerifyOpts {
-    algorithms?: string[]
-    audience?: string | RegExp | Array<string | RegExp>
-    clockTimestamp?: number
-    clockTolerance?: number
-    issuer?: string | string[]
-    ignoreExpiration?: boolean
-    ignoreNotBefore?: boolean
-    jwtid?: string
-    subject?: string
-    maxAge?: string
-  }
-
-  interface JWKSOpts {
-    cache?: boolean
-    rateLimit?: boolean
-  }
-
-  interface AuthenticOpts {
-    issWhitelist: string[]
-    jwks?: JWKSOpts
-    verify?: VerifyOpts
-  }
-
-  interface Validator {
-    (token: string): Promise<Authentic.JWT>
-  }
-
-  function Authentic(opts: AuthenticOpts): Validator
+  function Authentic(opts: Authentic.AuthenticOpts): Authentic.Validator
 
   namespace Authentic {
```

Comment:
I've manually pasted this file in the `360-toolbelt` project along with the `revocal` changes and it worked, but I've moved it below the namespace declaration, just in case.